tests/gl-cases/choropleth.js

```diff
@@ -0,0 +1,71 @@
+      }
+    }
+  ];
+
+  var myMap;
+
+  it('choropleth', function (done) {
+    imageTest.prepareImageTest();
+
+    var mapOptions = {
+      zoom: 4,
+      center: {
+        y: 33.60,
+        x: -95.00
+      }
+    };
+    myMap = common.createOsmMap(mapOptions, {}, true);
+
+    var scalarValues = states
+        .map(function (feature, index) {
+          return {
+            id: feature.properties.GEO_ID,
+            value: index
+          };
+        });
+
+    var layer = myMap.createLayer('feature', {features: ['choropleth']});
+
+    layer.createFeature('choropleth')
+         .data(states)
+         .scalar(scalarValues)
+         .choropleth({});
+    myMap.draw();
+
+    imageTest.imageTest('choropleth', 0.001, done, myMap.onIdle, 0, 2);
+  });
+  it('clean up', function () {
+    myMap.exit();
```

Comment:
Generally, I like to put setup and teardown inside `beforeEach` and `afterEach` groups to reinforce that individual tests shouldn't share state.